### PR TITLE
[NextPluggableDevice] Enable DtoD Copy ViaDMA

### PR DESCRIPTION
**Pull request by jzhoulon**
This PR is to enable DtoD copy ViaDMA support for NextPluggableDevice. This is needed when there are multiple PjRtDevice and send/recv op will use this registered copyfunc.

### CODE REVIEW

**Comment by jzhoulon** [comment]
@jyingl3 Can you help to have a look? thanks

**Comment by jyingl3** [comment]
Thanks for making this change Zhoulong! It looks good to me.

File: tensorflow/compiler/jit/pjrt_device_context.h [hunk 1]
```diff
@@ -52,6 +52,12 @@ class PjRtDeviceContext : public DeviceContext {
   bool use_pjrt_tensor_buffer_;
 };
 
+void PjRtDevice_DeviceToDeviceCopy(
```
[the remark this answers is not in the document]
Comment (jzhoulon) [reply tensorflow/compiler/jit/pjrt_device_context.h:55]: done

[the remark this answers is not in the document]
Comment (jyingl3) [reply tensorflow/compiler/jit/pjrt_device_context.h:55]: Thanks Zhoulong. It looks like the latest commit may not be uploaded?

[the remark this answers is not in the document]
Comment (jzhoulon) [reply tensorflow/compiler/jit/pjrt_device_context.h]: done, seem previously not push successfully, sorry for bring the trouble.

**Review by jyingl3 — APPROVED** [review]

File: tensorflow/compiler/jit/pjrt_device_context.h (hunk 1 above)
[the remark this answers is not in the document]
Comment (jyingl3) [reply tensorflow/compiler/jit/pjrt_device_context.h]: no worries at all!

**Pull request merged**